src/main/java/Duke.java

```diff
@@ -1,10 +1,147 @@
+                    System.out.println(dash);
+                }
+
+            } else if (input.length() > 6 && input.substring(0, 7).equals("delete ")) {
+                int index = -1;
+                if (!input.equals("delete ")) {
+                    index = Integer.parseInt(input.substring(7));
+                }
+                if (index > 0 && index <= inputList.size()) {
+                    System.out.println(dash);
+                    System.out.println("Pai Kia Bot: ok i delete this task chop chop: " + inputList.remove(index - 1).toString().substring(1));
+                    System.out.println(dash);
+                }
+
+            } else if (input.length() > 4 && input.substring(0, 5).equals("todo ")) {
+                if (!input.equals("todo ")) {
+                    String s = input.substring(5);
+                    CheckList c = new CheckList("T", s, false);
+                    inputList.add(c);
+                    System.out.println(dash);
+                    System.out.println("Pai Kia Bot: ok i added this task chop chop: " + c.toString().substring(1));
+                    System.out.println(dash);
+                } else {
+                    System.out.println(dash);
+                    System.out.println("Pai Kia Bot: no description leh, try again");
+                    System.out.println(dash);
+                }
+            } else if (input.length() > 5 && input.substring(0, 6).equals("event ")) {
+                if (!input.equals("event ")) {
+                    String[] s = input.substring(6).split("/at ");
+                    CheckList c = new CheckList("E", s[0] + "(at: " + s[1] + ")", false);
+                    inputList.add(c);
+                    System.out.println(dash);
+                    System.out.println("Pai Kia Bot: ok i added this task chop chop: " + c.toString().substring(1));
+                    System.out.println(dash);
+                } else {
+                    System.out.println(dash);
+                    System.out.println("Pai Kia Bot: no description leh, try again");
+                    System.out.println(dash);
+                }
+            } else if (input.length() > 8 && input.substring(0, 9).equals("deadline ")) {
+                if (!input.equals("deadline ")) {
+                    String[] s = input.substring(9).split("/by ");
+                    CheckList c = new CheckList("D", s[0] + "(by: " + s[1] + ")", false);
+                    inputList.add(c);
+                    System.out.println(dash);
+                    System.out.println("Pai Kia Bot: ok i added this task chop chop: " + c.toString().substring(1));
+                    System.out.println(dash);
+                } else {
+                    System.out.println(dash);
+                    System.out.println("Pai Kia Bot: no description leh, try again");
```

Comment:
I think this line `"Pai Kia Bot: no description leh, try again"` could be formatted as a String constant at the start of your code so that it can be reused throughout your program